### ISSUE TITLE
axlmp.exe not found when try to execute

### ISSUE THREAD

**Issue by PMtr** [icsharpcode/SharpDevelop#810]
OS : Windows 10 1703 (creatorsupdate).
Sharpdevelop 5.1 rev 5216
Hello,
I installed Sharpdevelop with .net 4.5.1 and windows SDK win7 for .net 4 as proposed on the web site.
and when I try to execute a bit of code, I get the error:
"""la tâche a échoué, car "AxImp.exe" est introuvable ou le Kit de développement Microsoft Windows SDK correct n'est pas installé. La tâche recherche "AxImp.exe" dans le sous-répertoire "bin" à l'emplacement spécifié par la valeur InstallationFolder de la clé de Registre HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v8.0A\WinSDK-NetFx40Tools-x86. Peut-être pouvez-vous résoudre le problème en effectuant l'une des opérations suivantes : 1.) Installer le Kit de développement Microsoft Windows SDK. 2.) Installer Visual Studio 2010. 3) Configurer manuellement la clé de Registre ci-dessus au bon emplacement. 4.) Passer le bon emplacement au paramètre "ToolPath" de la tâche. (MSB3091)"""

Sorry for the French language, to summarize axlmp.exe is not on my machine (I did a search all over the disk).
What to do to have it installed on my computer?
Must I installed windows SDK for build 1703?

Thanks for your help.

**Comment by geek1011** [comment]
Yes, you will need to install the Windows SDK for Windows 10 1703.

**Comment by PMtr** [comment]
Hi,
Thanks for this quick reply
I’ll try it
BR.

De : Patrick G [mailto:notifications@github.com]
Envoyé : vendredi 2 mars 2018 03:30
À : icsharpcode/SharpDevelop <SharpDevelop@noreply.github.com>
Cc : Marchand, Pascal <Pascal.Marchand@Honeywell.com>; Author <author@noreply.github.com>
Objet : [External] Re: [icsharpcode/SharpDevelop] axlmp.exe not found when try to execute (#810)


Yes, you will need to install the Windows SDK for Windows 10 1703.
